WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -4990,7 +4990,7 @@
 "migration.notifications.actions.secondary.title" = "Decide later";
 
 /* Footer for the migration notifications screen. */
-"migration.notifications.footer" = "When the alert apears tap Allow to continue receiving all your WordPress notifications.";
+"migration.notifications.footer" = "When the alert appears tap Allow to continue receiving all your WordPress notifications.";
```

Comment:
As mentioned in #19906, this will trigger a new translations upload when landing in `trunk`, but it should not disrupt the translation progress because GlotPress should identify this string as fuzzy.

I'll be checking the outcome later/tomorrow and communicate on the `.org` Slack if necessary.